api/support/api.go

```diff
@@ -0,0 +1,145 @@
+// Copyright 2017 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+// Package support contains the support service API client.
+package support
+
+import (
+	"bytes"
+	"encoding/json"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+
+	"github.com/juju/errors"
+	"github.com/juju/romulus/wireformat/support"
+	"gopkg.in/macaroon-bakery.v1/httpbakery"
+	"gopkg.in/macaroon.v1"
+)
+
+var DefaultURL = "https://api.jujucharms.com/omnibus/v2"
+
+// SupportAuthClient defines the interface available to clients of the support api.
```

Comment:
hmm.. support.SupportAuthClient.. stuttering.. can we have just support.AuthClient?